cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -92,7 +92,7 @@ class ReporterCommand : CliktCommand(
 
     private val reportFormats by option(
         "--report-formats", "-f",
-        help = "The list of report formats that is generated."
+        help = "The list of report formats that is generated, any of ${Reporter.ALL.map { it.reporterName }}."
```

Comment:
Is `joinToString()` needed here?

Reply:
Consider using just `','` instead of  `', '` as separator.

Reply:
I have used the default string representation for lists to be consistent with the package manager and scanner lists printed in the respective commands.